### PR TITLE
Feature/scroll navigation

### LINKED ISSUE #44: Scroll Navigation [Dev-12.0]
User story:
**As a** user
**I want** the ability to navigate articles by scrolling instead of swiping
**So that** I can choose a navigation style, according to my preference

Acceptance criteria:
_Given_  the issue navigation is reworked
_When_ scroll and swipe navigation are implemented
_Then_ the swiper.js library is used

_Given_ the issue navigation is reworked
_When_ swipe navigation is implemented
_Then_ swipe contains pagination

_Given_ the issue navigation is reworked
_When_ scroll navigation is implemented
_Then_ scroll does not contain pagination

Dependencies:
none

Description:
Users have repeatedly criticized the swipe function. Some like it though. Our idea: Introduce an scroll function in addition and let users decide (via a settings menu) what they prefer.

DoD Checklist:
- [ ] Acceptance criteria are met
- [ ] Test by developer has been done
- [ ] Code is commented if required
- [ ] Code/PR review has been done
- [ ] Feature is documented
- [ ] Internal test by a 2nd Dev or the PO has been done

### PR DESCRIPTION
**Pull request by hstaatsxx**
Closes #44 and #57

### CODE REVIEW

File: src/pages/SettingsPage.tsx [hunk 1]
```diff
@@ -21,9 +22,15 @@ export const SettingsPage: FC = () => {
     ctx.applyNewTheme(newTheme);
   }
 
+  const handleToggleSwipe = (newSwipe: SWIPE) => {
+    storeSwipe(newSwipe);
+    ctx.applyScollSet(newSwipe);
+    history.push('/')
```
Comment (DevJacobi) [inline src/pages/SettingsPage.tsx:28]: All the other options do not push us to the main menu when we update a setting. I think we should not do that here

**Review by DevJacobi — CHANGES_REQUESTED** [review]
Great work 🎉 
Just a minor remark

File: src/pages/Swiperview.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        }
+
+        fetchIssue();
+    }, [ctx, match.params.id]);
+
+
+    return (
+            <Swiper
+                direction={ swipe === SWIPE.VERTICAL ? 'vertical' : 'horizontal' }
+                pagination={{ clickable: true, enabled: swipe === SWIPE.HORIZONTAL ? true : false }} modules={[Pagination]}>
```
[the remark this answers is not in the document]
Comment (hstaatsxx) [reply src/pages/Swiperview.tsx]: okay, now it works. funny :D

**Review by DevJacobi — APPROVED** [review]

**Comment by DevJacobi** [comment]
Tested

https://user-images.githubusercontent.com/43578928/213171335-7ad9c25d-c6f6-4931-a2b2-8b9c9c30b9f6.mp4

**Pull request merged**